src/common/m_variables_conversion.fpp

```diff
@@ -518,6 +518,7 @@ contains
             G_K = 0d0
             do i = 1, num_fluids
                 !TODO: change to use Gs directly here?
+                !TODO: Make this changes as well for GPUs
```

Comment:
? what is purpose of these todos

Reply:
Mauro says they were there since the dawn of time. Someone just needs to take a look. The code works on NVIDIA GPUs....